daal4py/oneapi/__init__.py

```diff
@@ -33,5 +33,20 @@
     os.environ['PATH'] = current_path + os.pathsep + os.environ['PATH']
     os.environ['PATH'] = installed_package_path + os.pathsep + os.environ['PATH']
 
-from _oneapi import *
-from _oneapi import _get_sycl_ctxt, _get_device_name_sycl_ctxt, _get_sycl_ctxt_params
+try:
+    from _oneapi import *
+    from _oneapi import _get_sycl_ctxt, _get_device_name_sycl_ctxt, _get_sycl_ctxt_params
+except ModuleNotFoundError:
+    raise
```

Comment:
Why you need this?

Reply:
We use in examples this exception: https://github.com/intel/scikit-learn-intelex/blob/master/examples/run_examples.py#L61

For example, Mac Os don't build daal4py with dpcpp support, so daal4py.oneapi should throw ModuleNotFoundError